hue/hue/base/hue-db-init-job.yaml

```diff
@@ -0,0 +1,28 @@
+---
+apiVersion: batch/v1
```

Comment:
Thinking about this a bit more - what happens when we upgrade and the Jub is reconciled? How do we make it run again? (IIRC Job only runs once and there is no way to retrigger it other than delete and apply again)

Reply:
Gotcha. Good point. I think we should rather have it as an init container for Hue then. WDYT? :+1:

Reply:
Moved :+1: